src/pages/SchoolPage.tsx

```diff
@@ -81,6 +83,9 @@ const SchoolPage: FC<RouteComponentProps<{ schoolID: string }>> = ({ schoolID })
 
   const { school } = data as { school: ISchoolNode };
 
+  if (school.publicInstitutionData && school.publicInstitutionData.regon && isRegon(schoolID))
+    return <Redirect to={`/school/${school.schoolId}`} noThrow />;
```

Comment:
What is the type of this redirect? https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls#define-canonical